PhysicsTools/PatAlgos/test/BuildFile.xml

```diff
@@ -3,7 +3,6 @@
 <library name="PhysicsToolsPatAlgos_testAnalyzers" file="private/*.cc">
   <use name="FWCore/Framework"/>
   <use name="FWCore/ParameterSet"/>
-  <use name="PhysicsTools/PatUtils"/>
```

Comment:
if this and other BuildFile updates are not related to the (re)location of the BaseMVAValueMapProducer, please take them out and apply in a separate PR.
This would separate the straightforward updates from the (partial) impasse.

Reply:
No problem, I have factored out the other changes in https://github.com/cms-sw/cmssw/pull/31533